### PR TITLE
105272010 methods and recursion homework 2 master

### PR DESCRIPTION
**Pull request by Lucasjh**

### CODE REVIEW

File: methods_and_recursion/bottles_of_beer.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+ninetynine_beers = 99
+
+beer_on_the_wall = ninetynine_beers
+
+bot = 'bottles'
+
+while beer_on_the_wall > 2
```
Comment (jasonnoble) [inline methods_and_recursion/bottles_of_beer.rb]: This hw was for a slide deck about methods, can you pull out some methods?  Perhaps `sing_a_verse(number)` and `pluralize(string, number)`?

File: methods_and_flow_control/fizz_buzz.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
```
Comment (jasonnoble) [inline methods_and_flow_control/fizz_buzz.rb:1]: This should be in a separate branch....

File: methods_and_recursion/bottles_of_beer_recursive.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+
+# This a recursive version by DaVinci alumni that Jon found online.
+# I understand how it works, I did not write it. Bill helped me make it
+# better, using return instead of abort.
+# http://mark.kreyman.com/76/ruby/99-bottles-of-beer-ruby-methods-and-recursion/
+
+def pluralize(n, text)
+  return text + 's' if n != 1
```
Comment (jasonnoble) [inline methods_and_recursion/bottles_of_beer_recursive.rb]:
```
text += 's' if n != 1
"#{n} #{text}"
```

File: methods_and_recursion/bottles_of_beer_recursive.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  text
+end
+
+def bottles_on_the_wall(n)
```
Comment (jasonnoble) [inline methods_and_recursion/bottles_of_beer_recursive.rb]: `n` is a horrible variable name.

File: methods_and_flow_control/fizz_buzz.rb (hunk 2 above)
Comment (Lucasjh) [inline methods_and_flow_control/fizz_buzz.rb:1]: I see all four learning_by_doing subdirectories in this branch, why is the one .rb showing up in this commit?

File: methods_and_recursion/bottles_of_beer_recursive.rb (hunk 4 above)
Comment (Lucasjh) [inline methods_and_recursion/bottles_of_beer_recursive.rb]: changed to number_of_bottles

File: methods_and_recursion/bottles_of_beer_recursive.rb (hunk 3 above)
Comment (Lucasjh) [inline methods_and_recursion/bottles_of_beer_recursive.rb]: I tried this change and it broke the code

**Pull request merged**